src/renderer/library/components/settings/KeyboardSettings.tsx

```diff
@@ -389,11 +389,87 @@ class KeyboardSettings extends React.Component<IProps, IState> {
                 description: `${__("settings.keyboard.description.ToggleReaderFullscreenDesc")}`,
             },
         };
-        
+
 
         const filteredShortcuts = isSearchEmpty
-        ? ObjectKeys(sortObject(this.props.keyboardShortcuts) as TKeyboardShortcutsMap)
-        : ObjectKeys(cleanNames).filter(key => cleanNames[key].name.toLowerCase().includes(this.state.searchItem?.toLowerCase()));
+            ? ObjectKeys(sortObject(this.props.keyboardShortcuts) as TKeyboardShortcutsMap)
+            : ObjectKeys(cleanNames).filter(key => cleanNames[key].name.toLowerCase().includes(this.state.searchItem?.toLowerCase()));
+
+        const exportHtml = () => {
+            const element = document.getElementById("content-to-export");
+
+            if (element) {
+                let htmlContent = element.outerHTML;
+
+                const replaceSvgWithSpanName = (html: string): string => {
+                    const parser = new DOMParser();
+                    const doc = parser.parseFromString(html, "text/html");
+                    const spans = doc.querySelectorAll("span[title]");
+
+                    spans.forEach((span) => {
+                        const spanName = span.getAttribute("title") || "";
+                        span.innerHTML = `${spanName} + `;
+                    });
+
+                    return doc.body.innerHTML;
+                };
+
+                htmlContent = replaceSvgWithSpanName(htmlContent);
+
+
+                const getCssStyles = (): string => {
+                    let css = "";
+                    const styleSheets = document.styleSheets;
+
+                    for (let i = 0; i < styleSheets.length; i++) {
+                        const rules = styleSheets[i].cssRules;
+                        if (rules) {
+                            for (let j = 0; j < rules.length; j++) {
+                                css += rules[j].cssText;
+                            }
+                        }
+                    }
+                    return css;
+                };
+
+                const cssStyles = getCssStyles();
+
+                const completeHtml = `
+        <!DOCTYPE html>
```

Comment:
@arthur-lemeur  Is it really necessary to add this HTML export feature ? , I do not think this is what this PR is addressed. Moreover This is the job of the documentation!, there are no real benefits to maintain an html export. From my point of view.
If you really want to propose this feature, please create a new PR dedicated to this PR !

Reply:
Okay, I do not think exporting an html file to the local filesystem is the good way to do it. Why do not integrate it in the keyboard shortcut GUI presentation like a view/edit mode ? We have to keep it simple ! At the end it's a codebase that we have to maintain.

Reply:
We considered it easier to export an HTML that already exists in the GUI than a Print/PDF generator and template.

Reply:
A file need to be versioned, that means non-sens in this case when any keyboard shortcut can quickly be updated. What we can do instead it's to increase the user experience for the keyboard shortcut FROM thorium. 
Think that a file need to be localised, it's OS centric, can have many copy, can be stored in many place in the disk, and finally the user need to deal with it (open and load the html file in the browser), just for the purpose of a better keyboard shortcut presentation.

Reply:
We have to look from the electron API, perhaps we can try to open a new BrowserWindow attached to is parent.

Reply:
I reverted the commit and push to a new draft PR https://github.com/edrlab/thorium-reader/pull/2949 

Thanks @gautierchomel for your discussion, Please let's continue the discussion in the PR.